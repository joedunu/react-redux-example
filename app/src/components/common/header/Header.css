.header {
    background-color: beige;
    padding: 20px;
    font-family: "Arial Narrow";
    color: #636363;
}