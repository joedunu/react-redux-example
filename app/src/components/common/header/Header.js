import React from 'react'
import styles from './Header.css'
import { Link } from 'react-router-dom'

const Header = () => (
  <div className={styles.header}>
    <nav className='navbar navbar-toggleable-md navbar-light bg-faded'>
      <a className='navbar-brand' href='#'>React Redux Example</a>
      <div className='collapse navbar-collapse' id='navbarNav'>
        <ul className='navbar-nav mr-auto'>
          <li className='nav-item'>
            <Link className='nav-item nav-link' to={'/'}>Home</Link>
          </li>
          <li className='nav-item'>
            <Link className='nav-link' to={'/add-user'}>Add User</Link>
          </li>
          <li className='nav-item'>
            <Link className='nav-item nav-link' to={'/edit-user/1'}>Edit User</Link>
          </li>
        </ul>
      </div>
    </nav>
  </div>
)

export default Header
