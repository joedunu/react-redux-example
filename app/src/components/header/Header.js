import React from 'react'
import styles from './Header.css'

const Header = () => (
  <div className={styles.header}>
    <h1>React Redux Example</h1>
  </div>
)

export default Header
