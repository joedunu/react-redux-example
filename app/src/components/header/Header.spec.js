import React from 'react'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'
import Header from './Header'

test('Search snapshot test', () => {
  const component = shallow(<Header />)
  const tree = shallowToJson(component)
  expect(tree).toMatchSnapshot()
})
