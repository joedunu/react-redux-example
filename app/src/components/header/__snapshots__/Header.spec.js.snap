// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search snapshot test 1`] = `
<div>
  <h1>
    React Redux Example
  </h1>
</div>
`;
